static/js/components/LearnerSearch.js

```diff
@@ -99,6 +100,91 @@ export default class LearnerSearch extends SearchkitComponent {
 
   searchkitTranslations: Object = makeSearchkitTranslations();
 
+  renderSearchHeader: Function = (openEmailComposer: Function): React$Element<*>|null => {
+    if (_.isNull(this.getResults())) {
+      return null;
+    }
+
+    return (
+      <Grid className="search-header">
+        <Cell col={6} className="result-info">
+          <button
+            id="email-selected"
+            className="mm-button minor-action"
+            onClick={() => openEmailComposer(this.searchkit)}
+          >
+            Email These Learners
+          </button>
+          <HitsStats component={HitsCount} />
+        </Cell>
+        <Cell col={2} />
+        <Cell col={4} className="pagination-sort">
+          <SortingSelector options={sortOptions} listComponent={CustomSortingSelect} />
+          <Pagination showText={false} listComponent={CustomPaginationDisplay} />
+        </Cell>
+        <Cell col={12} className="mm-filters">
+          <SelectedFilters />
+          <ResetFilters component={CustomResetFiltersDisplay}/>
+        </Cell>
+      </Grid>
+    );
+  }
+
+  renderFacets: Function = (): React$Element<*> => {
+    if (_.isNull(this.getResults())) {
+      return (
+        <Card className="fullwidth" shadow={1}>
+          <div className="no-hits left-nav">
+            There are no users in the program.
```

Comment:
Is it possible to display the program name, to make this clearer? For example, if the program is named "Analog Learning", this message should say: "There are no users in the Analog Learning program."

If this is too difficult, we can at least make this clearer by changing the word "the" to the word "this", so that the message says: "There are no users in this program."